src/AzureAppConfigurationImpl.ts

```diff
@@ -657,3 +696,8 @@ function getValidFeatureFlagSelectors(selectors?: SettingSelector[]): SettingSel
         return getValidSelectors(selectors);
     }
 }
+
+function isFailoverableError(error: any): boolean {
+    return isRestError(error) && (error.code === "ENOTFOUND" ||

```

Comment:
Did you figure out the "ENOENT" error you met before?

Reply:
This error is thrown when dns file modified, I think it's not typical for failover scenario. If customer encounter it in the future, we can update the failoverable error then.

Reply:
Added 'ENOENT' to failvoerable error.